.azure-devops/templates/set-testenv-sentinel.yml

```diff
@@ -31,6 +31,7 @@ steps:
             "azext_dt_ep_eventhub_namespace": os.environ.get("AZEXT_DT_EP_EVENTHUB_NAMESPACE", sentinel_value),
             "azext_dt_ep_eventhub_policy": os.environ.get("AZEXT_DT_EP_EVENTHUB_POLICY", sentinel_value),
             "azext_dt_ep_eventhub_topic": os.environ.get("AZEXT_DT_EP_EVENTHUB_TOPIC", sentinel_value),
+            "azext_dt_ep_eventhub_queue": os.environ.get("AZEXT_DT_EP_EVENTHUB_QUEUE", sentinel_value),
```

Comment:
Future iteration comment - because these originally for digital twin endpoint resources can/are used across iot service areas, we should think about renaming them to something more generic rather than dt focused.

Reply:
Added a work item https://msazure.visualstudio.com/One/_sprints/taskboard/IoT-Platform-UPX/One/Custom/IoT/2208?workitem=15274556